From ff67a10492cd41d0b77ec238a1134b160818f3d2 Mon Sep 17 00:00:00 2001
From: Mohamad-Adwan <s1202827@stu.najah.edu>
Date: Tue, 15 Apr 2025 14:39:42 +0300
Subject: [PATCH 09/14] Donee

---
 src/config/dbConfig.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/config/dbConfig.ts b/src/config/dbConfig.ts
index 46c8dd0..80737e3 100644
--- a/src/config/dbConfig.ts
+++ b/src/config/dbConfig.ts
@@ -2,6 +2,6 @@
 // Database configuration
 // Replace these values with your actual database connection details
 export const dbConfig = {
-   apiUrl: 'http://localhost:3000/api', // API URL for your local Node.js server
-  // apiUrl: 'https://my-website-react-and-node-js-z1jd.onrender.com/api',
+  //  apiUrl: 'http://localhost:3000/api', // API URL for your local Node.js server
+  apiUrl: 'https://my-website-react-and-node-js-z1jd.onrender.com/api',
 };
-- 
2.45.1.windows.1

